pom.xml

```diff
@@ -74,6 +75,11 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-logging</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
Where `guava` is used by EE?

Reply:
It is already referenced indirectly and used by the e.g. AnalisysUtils:
com.google.common.io.Files.write

I have upgraded it because default version 20.0 contains Files marked as @Beta annotation.
So I switched to officially stable version by this upgrade

Reply:
I see, but why not to upgrade transitive dependency rather than point the library directly?